typescript/infra/config/environments/mainnet2/validators.ts

```diff
@@ -1,151 +1,157 @@
-import { CoreChainName } from '@hyperlane-xyz/sdk';
-
-import {
-  CheckpointSyncerType,
-  ValidatorBaseChainConfigMap,
-} from '../../../src/config/agent';
+import { ValidatorBaseChainConfigMap } from '../../../src/config/agent';
+import { Contexts } from '../../contexts';
+import { validatorsConfig } from '../utils';
 
 import { environment } from './chains';
 
-const s3BucketRegion = 'us-east-1';
-
-const s3BucketName = (chainName: CoreChainName, index: number) =>
-  `hyperlane-${environment}-${chainName}-validator-${index}`;
-
-export const validators: ValidatorBaseChainConfigMap = {
-  celo: {
-    interval: 5,
-    reorgPeriod: 0,
-    validators: [
-      {
-        address: '0x1f20274b1210046769d48174c2f0e7c25ca7d5c5',
-        name: s3BucketName('celo', 0),
-        checkpointSyncer: {
-          type: CheckpointSyncerType.S3,
-          bucket: s3BucketName('celo', 0),
-          region: s3BucketRegion,
+export const validatorChainConfig = (
+  context: Contexts,
+  count: number = 1,
+): ValidatorBaseChainConfigMap => {
+  return {
+    celo: {
+      interval: 5,
+      reorgPeriod: 0,
+      validators: validatorsConfig(
+        {
+          [Contexts.Hyperlane]: ['0x1f20274b1210046769d48174c2f0e7c25ca7d5c5'],
+          [Contexts.ReleaseCandidate]: [
+            '0xe7a82e210f512f8e9900d6bc2acbf7981c63e66e',
+          ],
         },
-      },
-    ],
-  },
-  ethereum: {
-    interval: 5,
-    reorgPeriod: 20,
-    validators: [
-      {
-        address: '0x4c327ccb881a7542be77500b2833dc84c839e7b7',
-        name: s3BucketName('ethereum', 0),
-        checkpointSyncer: {
-          type: CheckpointSyncerType.S3,
-          bucket: s3BucketName('ethereum', 0),
-          region: s3BucketRegion,
+        context,
+        environment,
+        'celo',
+        count,
+      ),
+    },
+    ethereum: {
+      interval: 5,
+      reorgPeriod: 20,
+      validators: validatorsConfig(
+        {
+          [Contexts.Hyperlane]: ['0x4c327ccb881a7542be77500b2833dc84c839e7b7'],
+          [Contexts.ReleaseCandidate]: [
+            '0xaea1adb1c687b061e5b60b9da84cb69e7b5fab44',
+          ],
         },
-      },
-    ],
-  },
-  avalanche: {
-    interval: 5,
-    reorgPeriod: 3,
-    validators: [
-      {
-        address: '0xa7aa52623fe3d78c343008c95894be669e218b8d',
-        name: s3BucketName('avalanche', 0),
-        checkpointSyncer: {
-          type: CheckpointSyncerType.S3,
-          bucket: s3BucketName('avalanche', 0),
-          region: s3BucketRegion,
+        context,
+        environment,
+        'ethereum',
+        count,
+      ),
+    },
+    avalanche: {
+      interval: 5,
+      reorgPeriod: 3,
+      validators: validatorsConfig(
+        {
+          [Contexts.Hyperlane]: ['0x4c327ccb881a7542be77500b2833dc84c839e7b7'],
```

Comment:
Should this hyperlane addresses match the ones you're updating, '0xa7aa52623fe3d78c343008c95894be669e218b8d' here?